### DISCUSSION TITLE
Use Mantine Input component for React-Select

### DISCUSSION THREAD

**Discussion by xoitx** [mantinedev/mantine#3914] [Q&A] [answered]
How do I use the mantine `<Input/>` component instead of React-Select's
I need the input and the options to be styled according to Dark or Light theme

I tried replacing the component with Input, but it breaks the page display

Also does Mantine multiselect use React-Select under the hood?
So all of React-Select props would be applicable?

Any suggestions?

**Comment by rtivital** [comment, marked as answer]
I doubt that it is possible. You can use Mantine Select or MultiSelect components.

> Also does Mantine multiselect use React-Select under the hood?


No
